tests/rt.sh

```diff
@@ -305,6 +305,34 @@ elif [[ $MACHINE_ID = jet.* ]]; then
   cp fv3_conf/fv3_slurm.IN_jet fv3_conf/fv3_slurm.IN
   cp fv3_conf/compile_slurm.IN_jet fv3_conf/compile_slurm.IN
 
+elif [[ $MACHINE_ID = s4.* ]]; then
+
+  module load rocoto/1.3.2
+  ROCOTORUN=$(which rocotorun)
+  ROCOTOSTAT=$(which rocotostat)
+  ROCOTOCOMPLETE=$(which rocotocomplete)
+  ROCOTO_SCHEDULER=slurm
+
+  PYTHONHOME=/opt/miniconda/3.8-s4
+  export PATH=$PYTHONHOME/bin:$PATH
+  export PYTHONPATH=$PYTHONHOME/lib/python3.8/site-packages
+  ECFLOW_START=
```

Comment:
Is it possible to configure `ecflow` in addition to `rocoto`?

Reply:
I've added support for `ecflow` to rt.sh and I did a quick test using it.  I'll do a full set of tests with it as well.